CONTRIBUTING.md

```diff
@@ -205,6 +205,8 @@ example would explicitly use `Timeout::new`. For example:
 /// # }
 ```
 
+To reduce the effort required to review documentation-related changes, `tracing`'s CI system generates preview websites containing the `rustdoc` output for the entire repository for each PR.
```

Comment:
Nit: can this be wrapped to match the rest of this file?

Reply:
Can we add instructions on how to actually view the docs previews here? Like:
> The preview can be viewed by clicking the `details` link on the `netlify/tracing-rs/deploy-preview` check on pull requests.

Reply:
fixed.